### PR TITLE
Improve startup time by prelocating libjemalloc

### PR DESCRIPTION
**Pull request by thobbs**
This cuts about 1s off of Cassandra's startup time, which should save us ~15m on the dtests.

This requires https://github.com/pcmanus/ccm/pull/519 in order to work
correctly.

### CODE REVIEW

**Comment by ptnapoleon** [comment]
+1

File: dtest.py [hunk 1]
```diff
@@ -115,6 +115,26 @@
...
+# Determine the location of the libjemalloc jar so that we can specify it
+# through environment variables when start Cassandra.  This reduces startup
+# time, making the dtests run faster.
+def find_libjemalloc():
+    if is_win():
+        # let the normal bat script handle finding libjemalloc
+        return ""
+
+    this_dir = os.path.dirname(os.path.realpath(__file__))
+    script = os.path.join(this_dir, "findlibjemalloc.sh")
```
Comment (ptnapoleon) [inline dtest.py]: Did you need to commit this script?

File: dtest.py [hunk 2]
```diff
@@ -115,6 +115,26 @@
...
+    p = subprocess.Popen([script], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    stdout, stderr = p.communicate()
+    if stderr or not stdout:
+        return "-"  # tells C* not to look for libjemalloc
```
Comment (ptnapoleon) [inline dtest.py]: The error handling here is less than perfect, as if this fails for certain reasons, it 100% explodes everywhere

**Comment by thobbs** [comment]
Damnit, forgot to include a new script file in the commit.  I'll open a new pull request.

File: dtest.py (hunk 2 above)
Comment (thobbs) [inline dtest.py]: I'll make that a bit more robust.

**Pull request merged**